schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,163 @@
+              },
+              "installAutotracking": {
+                "description": "Whether enable automatic tracking of install event.",
+                "type": "boolean"
+              },
+              "exceptionAutotracking": {
+                "description": "Whether enable crash reporting.",
+                "type": "boolean"
+              },
+              "diagnosticAutotracking": {
+                "description": "Whether enable tracker diagnostic.",
+                "type": "boolean"
+              }
+            }
+          },
+          "subjectConfiguration": {
+            "description": "Represents the configuration of the subject. The SubjectConfiguration can be used to setup the tracker with the basic information about the user and the app which will be attached on all the events as contexts.",
+            "type": "object",
+            "properties": {
+              "userId": {
+                "description": "The custom user identifier.",
+                "type": "string"
+              },
+              "networkUserId": {
+                "description": "The custom network user identifier.",
+                "type": "string"
+              },
+              "domainUserId": {
+                "description": "The custom domain user identifier.",
+                "type": "string"
+              },
+              "useragent": {
+                "description": "The custom user-agent. It overrides the user-agent used by default.",
+                "type": "string"
+              },
+              "ipAddress": {
+                "description": "The IP address (not automatically set).",
+                "type": "string"
+              },
+              "timezone": {
+                "description": "Override the timezone string set by the tracker.",
+                "type": "string"
+              },
+              "language": {
+                "description": "Override the language string set by the tracker.",
+                "type": "string"
+              }
+            }
+          },
+          "sessionConfiguration": {
+            "description": "Represents the configuration of a Session object which gets appended to each event sent from the Tracker and changes based on the timeout set for the inactivity of app when in foreground or background.",
+            "type": "object",
+            "properties": {
+              "backgroundTimeout": {
+                "description": "The amount of time that can elapse before the session id is updated while the app is in the background.",
+                "type": "integer",
+                "minimum": 0
+              },
+              "foregroundTimeout": {
+                "description": "The amount of time that can elapse before the session id is updated while the app is in the foreground.",
+                "type": "integer",
+                "minimum": 0
+              }
+            }
+          }
+        },
+        "required": [
+          "namespace"
```

Comment:
Is `networkConfiguration` not required?

Reply:
it's not required. If you don't set the networkConfiguration in the configuration bundle, the tracker will use the related namespace to remove the tracker instance if it exists. It's an implicit and rudimental way to clean instances.

Reply:
In that case, all looks good!